azext_iot/central/_help.py

```diff
@@ -1752,3 +1754,230 @@ def _load_central_edge_help():
+          --at {attestation}
+          --display-name {displayName}
+          --type {type}
+          --enabled true
+    """
+
+    helps[
+        "iot central enrollment-group verify-x509"
+    ] = """
+    type: command
+    long-summary: |
+                    Verify the primary or secondary x509 certificate of an enrollment group
+                    by providing a certificate with the signed verification code.
+    examples:
+      - name: Verify the primary or secondary x509 certificate of an enrollment group
+        text: >
+          az iot central enrollment-group verify-x509
+          --app-id {appid}
+          --id {enrollmentGroupId}
+          --entry {certificateEntry}
+          --certificate	{certificate}
+    """
+
+    helps[
+        "iot central enrollment-group verification-code"
+    ] = """
+    type: command
+    short-summary: Generate a verification code for the primary or secondary x509 certificate of an enrollment group
+    examples:
+      - name: Generate a verification code for the primary or secondary x509 certificate of an enrollment group
+        text: >
+          az iot central enrollment-group verification-code
+          --app-id {appid}
+          --id {enrollmentGroupId}
+          --entry {certificateEntry}
+    """
+
+
+def _load_central_scheduled_job_help():
+    helps[
+        "iot central scheduled-job"
+    ] = """
+          type: group
+          short-summary: Manage and configure IoT Central schedule job
+      """
+
+    helps[
+        "iot central scheduled-job list"
+    ] = """
+    type: command
+    short-summary: Get the list of scheduled job definitions in an application
+    examples:
+      - name: Get the list of scheduled job definitions in an application
+        text: >
+          az iot central scheduled-job list
+          --app-id {appid}
+    """
+
+    helps[
+        "iot central scheduled-job show"
+    ] = """
+    type: command
+    short-summary: Get details about a scheduled job by ID
+    examples:
+      - name: Get details about a scheduled job by ID
+        text: >
+          az iot central scheduled-job show
+          --app-id {appid}
+          --id {scheduledJobId}
+    """
+
+    helps[
+        "iot central scheduled-job delete"
+    ] = """
+    type: command
+    short-summary: Delete an existing scheduled job by ID
+    examples:
+      - name: Delete an existing scheduled job by ID
+        text: >
+          az iot central scheduled-job delete
+          --app-id {appid}
+          --id {scheduledJobId}
+    """
+
+    helps[
+        "iot central scheduled-job create"
+    ] = """
+    type: command
+    short-summary: Create a scheduled job by ID
+    examples:
+      - name: Create a schedule job with name
+        text: >
+          az iot central scheduled-job create
+          --app-id {appid}
+          --job-id {jobId}
+          --group-id {groupId}
+          --job-name {displayName}
+          --schedule	{jobScheduleJSONPatch}
```

Comment:
what happened to spacing here

Reply:
emm idk why flake8 or pylint didn't catch this. I'll fix it.